adding-behavior/changing-the-dom.md

```diff
@@ -0,0 +1,255 @@
+
+How can Keira accomplish her goal? She'll use JavaScript and her knowledge of the DOM to help her!
+
+To change elements in the DOM, we'll need to learn two things:
+
+- How to select HTML elements in the DOM
+- How to specify the changes we want
+
+## HTML Data in `document`
+
+To work with the DOM, we need to learn how to access the DOM. To do that, let's run an experiment:
+
+1. With _any_ website open, open the browser Dev Tools
+1. Open the console
+1. Run the statement `document`
+
+We should see the details of a document object print to the console.
+
+![The Crab Fan Site, and the Dev Tools console open. The Dev Tools console has output details about the document object.](../assets/adding-behavior_changing-the-dom_document.png)
+
+The browser provides the global variable `document`. This variable represents the root node of the DOM.
+
+The DOM represents all the HTML elements on a page. Because `document` is the root node of the DOM, we will use `document` to access and select other HTML elements.
+
+Whenever we want to access the DOM, we'll being with `document`.
+
+## Selecting Elements Using `document`
+
+Before we can change a specific element, we need to select that element first.
+
+The `document` object has a number of methods that allow us to select specific HTML elements.
+
+For example, the method `getElementById` will give us back an HTML element with a matching ID, or `null` if there are no matching elements.
+
+```js
+const fancyParagraphElement = document.getElementById("fancy-paragraph");
+```
+
+The above line initializes a `const` variable named `fancyParagraphElement`. It's set to an HTML element that has the idea `fancy-paragraph`.
+
+Here are a few built-in and available methods that operate off of `document`, which select HTML elements:
+
+| Method                   | Description                                                              |
+| ------------------------ | ------------------------------------------------------------------------ |
+| `getElementById`         | Returns the element that has the ID attribute with the specified value   |
+| `getElementsByClassName` | Returns a NodeList containing all elements with the specified class name |
+| `getElementsByTagName`   | Returns a NodeList containing all elements with the specified tag name   |
+| `querySelector`          | Returns the first element that matches the given CSS selector            |
+| `querySelectorAll`       | Returns a list containing all elements that match the given CSS selector |
+
+When we get back a list from `getElementsByClassName` `getElementsByTagName`, or `querySelectorAll`, we'll need to access elements through indexing or loops!
+
+## Modifying Elements
+
+Once we have our targeted HTML element, we can modify it!
```

Comment:
Just suggesting a clarifying what "targeted" means

```suggestion
Once we have our targeted, i.e. selected, HTML element, we can modify it!
```

Reply:
I'd go all the way of replacing targeted with selected. We already used the term selected previously, and I don't see a compelling reason to introduce targeted here
```suggestion
Once we have our selected HTML element, we can modify it!
```